new design #66

RocktimSaikia opened the pull request:
Hi in this pull request I made few changes in the overall home page design .All the designs specs is based on the issue [#66](https://github.com/JasonFritsche/BreweryFinder/issues/66) that I opened few days ago.

So again these are the things i worked on :
-  Got rid of the heavy background image that was not optimized  instead of went for a more simple flat design with a simple vector png which made the loading time for the page more faster.
- Used some simple basic css design principles to make the web page look more professional and standard( like neon gradients ,flat design color palettes ,standard google fonts).
- Made the footer stick to bottom and inside the footer added a contribution button linking to this repo.

**here is the final result :**
![brewer](https://user-images.githubusercontent.com/33410545/56075668-0f443680-5d7b-11e9-9750-275cc331b152.jpg)

Please let me know what you guys think of these chnages .

cdrani commented on src/App.css:
```diff
@@ -1,59 +1,99 @@
-@import url("https://fonts.googleapis.com/css?family=Kreon|Quicksand");
+@import url('https://fonts.googleapis.com/css?family=Kreon|Quicksand|Pacifico|Lobster|Raleway:600,800');
 
```
Font loading takes some time to load, especially fonts that aren't native to the browser. Are these new fonts necessary/integral to the design? If so, then the old ones should be removed. I recommend only 2-3 fonts.

cdrani commented on src/App.css:
```diff
@@ -1,59 +1,99 @@
   background-repeat: no-repeat;
-  background-size: cover;
+  background-size: cover; */
 }
```
There's already a body selector in `src/index.css`. Perhaps move the above css into `src/index.css`?

cdrani commented on src/App.css:
```diff
@@ -1,59 +1,99 @@
   animation: glow 1s ease-in-out infinite alternate;
-}
+} */
 
```
Please prefer classes over ids to reduce specificity as it's unneeded here. Could you also add a new line between each selector to space them out? Additionally remove the commented out css.

cdrani commented on src/components/BrewerySearch.js:
```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types'
 import RadioButtonContainer from './RadioButtonContainer'
+const img = require('../public/cheers.png')
+const githublogo = require('../public/github-logo.png')
 
```
Please use esmodule `imports` over commonjs `require` for uniformity.

RocktimSaikia commented:
@cdrani Thanks for the review i will make your requested changes asap 😊

RocktimSaikia commented:
@cdrani hi i made the requested changes that you mentioned. So again here is the overview of the new changes i made :
  - Removed the unused old google fonts that were taking time to load.
  - Moved the body selector css part to ```index.css``` file  .
  - Used classes over ids and added newline between every css selector
  - Replaced commonjs require with es6 modules.
  - Fixed the gradient background issue that was making the ``` go back button``` hard to read. 

A few new changes i made : 
  - Changed the background color of search input field from gradient to plain white.Which looks more clean now. 
  - Added gradient background to cards and made the card buttons block level which looks much better now.

Here is the changed homepage with the white colored input field  :
![new site](https://user-images.githubusercontent.com/33410545/56090940-5d316b00-5e5d-11e9-8008-1b5d53ac9c2b.jpg)

Search result page with new gradient cards : 
![bak](https://user-images.githubusercontent.com/33410545/56091038-666f0780-5e5e-11e9-8397-174a57e39a26.jpg)

JasonFritsche commented:
@RocktimSaikia Thanks for your contributions to the project. This is an awesome addition.

RocktimSaikia commented:
I am glad i could use this holiday season working on open source rather than my college stuff :)

JasonFritsche commented:
Feel free to make more PR's. If you see a bug or feature that is not in the current set of issues then feel free to open a new issue. Thanks!
